### PR TITLE
Fix description of Adadelta formula 

### PR DESCRIPTION
**Pull request by minhduc0711**
*Description of changes:*
I attempted to correct the description for the formula of the rescaled gradient. 
image

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_optimization/adadelta.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ $$\begin{aligned}
...
+The difference to before is that we perform updates with the rescaled gradient $\mathbf{g}_t'$ which is computed by multiplying $\mathbf{g}_t}$ with the square root of the ratio between the average second moment of the rate of change and the average second moment of the gradient. The use of $\mathbf{g}_t'$ is purely for notational convenience. In practice we can implement this algorithm without the need to use additional temporary space for $\mathbf{g}_t'$. As before $\epsilon$ is a parameter ensuring nontrivial numerical results, i.e., avoiding zero step size or infinite variance. Typically we set this to $\epsilon = 10^{-5}$.
```
Comment (minhduc0711) [inline chapter_optimization/adadelta.md:17]:
```suggestion
The difference to before is that we perform updates with the rescaled gradient $\mathbf{g}_t'$ which is computed by multiplying $\mathbf{g}_t$ with the square root of the ratio between the average second moment of the rate of change and the average second moment of the gradient. The use of $\mathbf{g}_t'$ is purely for notational convenience. In practice we can implement this algorithm without the need to use additional temporary space for $\mathbf{g}_t'$. As before $\epsilon$ is a parameter ensuring nontrivial numerical results, i.e., avoiding zero step size or infinite variance. Typically we set this to $\epsilon = 10^{-5}$. 
```

**Comment by mli** [comment]
Job d2l-en/PR-1180/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1180/

**Comment by goldmermaid** [comment]
Hi @minhduc0711 , thanks for the fixes. Actually there is a minor typo in the formula. I updated my modifications recently at https://github.com/d2l-ai/d2l-en/commit/09213267fa584a8dfab5cf750630e7bc63374ed9. Could you incorporate my changes to your PR as well? Thank you!

**Comment by minhduc0711** [comment]
Hi @goldmermaid, seems like your updates have made my original fix invalid, so I only changed a few words in your fix (minor phrasing stuff). Can you check if it's alright? Thanks.

**Comment by goldmermaid** [comment]
Hi @minhduc0711 , your fixes won't be invalid. Feel free to grab my changes if they make sense to you!

**Comment by mli** [comment]
Job d2l-en/PR-1180/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1180/

**Comment by mli** [comment]
Job d2l-en/PR-1180/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1180/

**Comment by minhduc0711** [comment]
@goldmermaid sorry, I meant that since you updated the formulas, the earlier "error" I found kinda made sense now.
There's only one minor thing I spotted: there's a description for `delta x_{t-1}` but formula 11.9.4 computes `delta x_t`, and I think this might cause a little bit of confusion.

**Comment by goldmermaid** [comment]
> @goldmermaid sorry, I meant that since you updated the formulas, the earlier "error" I found kinda make sense now.
> There's only one minor thing I spotted: there's a description for `delta x_{t-1}` but formula 11.9.4 computes `delta x_t`, and I think this might cause a little bit of confusion.

Good catch! I updated it again. If you are making the changes, i will post a new PR about it. Would you like to help me review that PR?

**Comment by minhduc0711** [comment]
Closed due to newer updates at #1196

**Pull request closed without merging**